testkit/src/main/scala/org/bitcoins/testkit/core/gen/CreditingTxGen.scala

```diff
@@ -230,6 +287,42 @@ sealed abstract class CreditingTxGen {
   def randoms: Gen[Seq[BitcoinUTXOSpendingInfo]] =
     Gen.choose(min, max).flatMap(n => Gen.listOfN(n, random))
 
+  private def computeAllTrueConditionalPath(
```

Comment:
Any way to make this better by just taking in a `UTXOSpendingInfo`?

Reply:
Sadly no, because we make the nested SpendingInfo's into `val`s so there would be errors if we did not construct the conditional path ahead of time